cmd/fyne_demo/tutorials/data.go

```diff
@@ -60,6 +56,18 @@ var (
 			"A container that provides scrolling for it's content.",
 			makeScrollTab,
 		},
+		"apptabs": {"AppTabs",
```

Comment:
This was in alphabetical order within the section, so if keeping the tabs subgroup I think this and doctabs belong below it, if it's being removed then they belong further up in the list (in previous position)

Reply:
Done